src/Host/Client/Impl/rtvs/R/util.R

```diff
@@ -183,13 +183,13 @@ export_to_image <- function(device_id, plot_id, device, width, height, resolutio
 }
 
 # Helper to export current plot to pdf
-export_to_pdf <- function(device_id, plot_id, width, height) {
+export_to_pdf <- function(device_id, plot_id, pdf_device, width, height, paper) {
```

Comment:
This is tested in IdeGraphicsDeviceTest (see ExportToPdfAsync), so you'll need to make some update there as well.